Sources/docc/DocCDocumentation.docc/adding-structure-to-your-documentation-pages.md

```diff
@@ -141,9 +141,14 @@ default organization and provide a more appropriate structure for your symbols.
 
 ![A screenshot showing the rendered documentation containing three topic groups: Creating a Sloth, Activities, and Schedule.](4_topics_2)
 
-To add an extension file to your documentation catalog for a specific symbol, use a text editor to create a new file named `Extension.md`.
-
-In the `Extension.md` file, replace the `Symbol` placeholder 
+To add an extension file for a specific symbol, use a text editor to create an
+`.md` file within the documentation catalog. If you're working in Xcode, you can
+select the documentation catalog in the project navigator and run the File ->
+New -> File From Template... menu command. Then click "Extension File" under the
+Documentation section. DocC ignores file names of documentation extensions; you
+can choose any file name you would like as long as it uses an `.md` extension.
```

Comment:
There are 3 pieces of information in this paragraph:
- General instruction to create a new markdown file
- Xcode specific detailed steps to create a new markdown file with the extension file template 
- General information about how developers can name the markdown file.

With the Xcode specific information in the middle and being the longest part of the paragraph, some developers might skip to the next paragraph and miss the general information about naming the markdown file. 

I think the solution to that is to split out either the Xcode specific informatin or the file naming information into an aside.

Reply:
Great idea - added a new aside.